--- src/load-config.js	2024-12-20 18:01:39.312760450 +0000
+++ src/load-config.js_patched	2024-12-20 18:02:25.796884483 +0000
@@ -10,7 +10,7 @@
 debug(`project root: '${projectRoot}'`);
 export const sqlDir = path.join(projectRoot, 'src', 'sql');
 
-let defaultDataDir = fs.existsSync('/data') ? '/data' : projectRoot;
+let defaultDataDir = '/var/lib/actual';
 
 if (process.env.ACTUAL_DATA_DIR) {
   defaultDataDir = process.env.ACTUAL_DATA_DIR;
