--- packages/sync-server/src/load-config.js	2025-02-12 10:15:34.000000000 +0100
+++ packages/sync-server/src/load-config.js.patched	2025-02-12 10:52:43.885120960 +0100
@@ -13,7 +13,7 @@
 debug(`project root: '${projectRoot}'`);
 export const sqlDir = path.join(projectRoot, 'src', 'sql');
 
-let defaultDataDir = fs.existsSync('/data') ? '/data' : projectRoot;
+let defaultDataDir = '/var/lib/actual';
 
 if (process.env.ACTUAL_DATA_DIR) {
   defaultDataDir = process.env.ACTUAL_DATA_DIR;
